addons/assigngear/loadouts/blu_f.hpp

```diff
@@ -51,6 +54,7 @@ class r : baseMan
     headgear[] = {"H_HelmetB","H_HelmetSpecB"};
     backpack[] = {"B_Kitbag_rgr"};
     primaryWeapon[] = {"arifle_MX_F"};
+    primaryMagazine[] = {"30Rnd_65x39_caseless_mag"};
```

Comment:
Won't this cause problems with inheritance, such as for the Automatic Rifleman?

Reply:
Good call, I'll sus it out.

Reply:
Fixed.

Reply:
Seems like you fixed it for the example loadout, but not here. You'll probably want to apply the change to fia, ind, and opf loadouts.

Reply:
Ah, true.

Reply:
Got them now.